tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -284,6 +286,52 @@ def test_slurm_protected_mode(
     _test_recover_from_protected_mode(pending_job_id, pcluster_config_reader, bucket_name, cluster, scheduler_commands)
 
 
+@pytest.mark.usefixtures("region", "os", "instance", "scheduler")
+@pytest.mark.fast_capacity_failover
+def test_fast_capacity_failover(
+    pcluster_config_reader,
+    clusters_factory,
+    test_datadir,
+    scheduler_commands_factory,
+):
+    cluster_config = pcluster_config_reader()
+    cluster = clusters_factory(cluster_config)
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    clustermgtd_conf_path = _retrieve_clustermgtd_conf_path(remote_command_executor)
+    scheduler_commands = scheduler_commands_factory(remote_command_executor)
+    # after the cluster is launched, apply the override patch to launch ice nodes
+    remote_command_executor.run_remote_script(str(test_datadir / "overrides.sh"), run_as_root=True)
+    nodes_in_scheduler = scheduler_commands.get_compute_nodes("queue1")
+    static_nodes, dynamic_nodes = get_partition_nodes(nodes_in_scheduler)
+    ice_dynamic_nodes = [node for node in dynamic_nodes if "ice-compute-resource" in node]
+    ice_static_nodes = [node for node in static_nodes if "ice-compute-resource" in node]
```

Comment:
These are not really ICE static nodes because they are launched successfully since the override was not present.  naming them ice_static_nodes makes me think you are expecting an ICE failure